packages/typescript/rules-snapshot.json

```diff
@@ -110,16 +112,26 @@
   "no-dupe-keys": "off",
   "no-empty-function": "off",
   "no-extra-parens": "off",
+  "no-extra-semi": "off",
+  "no-func-assign": "off",
+  "no-import-assign": "off",
   "no-magic-numbers": "off",
   "no-new-symbol": "off",
+  "no-obj-calls": "off",
   "no-redeclare": "off",
+  "no-setter-return": "off",
   "no-this-before-super": "off",
   "no-undef": "off",
   "no-unreachable": "off",
+  "no-unsafe-negation": "off",
   "no-unused-expressions": "off",
   "no-unused-vars": "off",
   "no-use-before-define": "off",
   "no-useless-constructor": "off",
+  "no-var": "error",
```

Comment:
These four seem new :thinking: These aren't even TypeScript rules. What's going on here?

Reply:
These come from the `plugin:@typescript-eslint/eslint-recommended` config ([source](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/src/configs/eslint-recommended.ts)), which is extended by `plugin:@typescript-eslint/recommended` (https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/src/configs/recommended.ts)).

They discuss it in their readme here: https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin/src/configs#eslint-recommended

Reply:
Got it, thanks. Makes sense.